src/Controller/RulesController.php

```diff
@@ -13,16 +13,5 @@
  * Returns responses for Rules routes.
  */
 class RulesController extends ControllerBase {
```

Comment:
You were more or less proposing to keep it even if it's now empty. I've no strong opinion on this one. For now, we can remove it. What do you think ?